PROJECT

```diff
@@ -9,8 +9,8 @@ resources:
   domain: cluster.x-k8s.io
   group: infrastructure
   kind: CloudStackCluster
-  path: sigs.k8s.io/cluster-api-provider-cloudstack/api/v1beta1
-  version: v1beta1
+  path: sigs.k8s.io/cluster-api-provider-cloudstack/api/v1beta3
+  version: v1beta3
```

Comment:
If we want to use the scaffolding tooling I think this is needed. I generally separate from the scaffolding after creating the initial project because I find its too rigid in its expectations.

I don't mind either way.